iceoryx_posh/source/popo/ports/interface_port.cpp

```diff
@@ -19,38 +19,38 @@ namespace iox
 {
 namespace popo
 {
-InterfacePort::InterfacePort(InterfacePortData* const f_member)
-    : BasePort(f_member)
+InterfacePort::InterfacePort(InterfacePortData* const interfacePortDataPtr) noexcept
+    : BasePort(interfacePortDataPtr)
 {
 }
 
-bool InterfacePort::dispatchCaProMessage(const capro::CaproMessage& f_message)
+bool InterfacePort::dispatchCaProMessage(const capro::CaproMessage& message) noexcept
 {
-    bool returner = getMembers()->m_caproMessageFiFo.push(f_message);
+    const bool returner = getMembers()->m_caproMessageFiFo.push(message);
```

Comment:
I'm quite sure this isn't a returner ;)
https://www.dict.cc/?s=returner

How about retVal or pushResult?

Reply:
+1 for pushResult

Reply:
You're right. Seems that I have to go the extra mile for adding the const ;-)